app/presenters/tree_builder_ops_rbac_features.rb

```diff
@@ -0,0 +1,160 @@
+class TreeBuilderOpsRbacFeatures < TreeBuilder
+  has_kids_for Menu::Section,     [:x_get_tree_section_kids]
+  has_kids_for Menu::Item,        [:x_get_tree_item_kids]
+  has_kids_for MiqProductFeature, [:x_get_tree_feature_kids]
+
+  attr_reader :role, :features, :editable, :root_counter
+
+  def initialize(name, type, sandbox, build, role:, editable: false)
+    @role     = role
+    @editable = editable
+    @features = @role.miq_product_features.order(:identifier).pluck(:identifier)
+
+    @root_counter = []
+
+    # Make sure tree_state doesn't hold on to old data between requests
+    TreeState.new(sandbox).remove_tree(name)
```

Comment:
The idea of the `TreeState` was to have an interface class between the trees and the session (`@sb`).

That got lost somehow as more people worked on this.

The long shot was to have the `TreeBuilder` classes work w/o the `session` and be testable w/o the `session`.

It would be nice to get back to that idea.

It this particular place we can leave this line as it is I think. We should fix this in a separate PR.